source/_integrations/loqed.markdown

```diff
@@ -0,0 +1,60 @@
+---
+title: Loqed Touch Smart Lock
+description: Instructions on how to integrate a Loqed Touch Smart Lock
+ha_category:
+  - Lock
+ha_release: 2023.7
+ha_iot_class: Local Push
+ha_codeowners:
+  - "@mikewoudenbergs"
+ha_domain: loqed
+ha_platforms:
+  - lock
+  - sensor
+ha_config_flow: true
+ha_dhcp: true
+ha_integration_type: integration
+---
+
+Integrate your LOQED Touch Smart Lock with Home Assistant. The lock instantly notifies Home Assistant of a lock state change and you can change the lock state yourself.
+
+## Features
+
+This integration supports:
+
+- Send real-time status changes of the lock (open, unlock, lock)
+- Change the lock state (open, unlock, lock).
+  - Only if your lock has a fixed knob on the outside of your door, you can use the “open” lock state. If you do not have this (thus you have a handle on the outside of your door that you can push down), this command will behave as if the unlock command is sent.
+
+## Prerequisites
+
+On the [LOQED personal access token website](https://integrations.production.loqed.com/personal-access-tokens), please follow the following steps:
+
+{% details "Generate access token" %}
+
+1. Login with your LOQED App e-mail address (you need to be admin)
+2. Tap on "Create"
+3. Give your personal access token a name (this will not be used further on, but we recommend something like "Home Assistant" to be able to recognize it as used by Home Assistant)
+4. Tap on Save
+5. Store the access token somewhere you can easily copy/paste from as you'll need them in the next step (and it will only be shown once). Note that you can use this token for setting up multiple locks.
+   {% enddetails %}
```

Comment:
```suggestion
1. Login with your LOQED App e-mail address (you need to be an admin).
2. Select **Create**.
3. Give your personal access token a name (this will not be used further on, but we recommend something like "Home Assistant" to recognize it as used by Home Assistant).
4. Select **Save**.
5. Store the access token somewhere you can easily copy/paste from, as you'll need them in the next steps (and it will only be shown once). Note: that you can use this token for setting up multiple locks.
   {% enddetails %}
```